modules/asr/sensory/sensory_asr.c

```diff
@@ -130,13 +130,8 @@ asr_error_t asr_process(asr_port_t *ctx, int16_t *audio_buf, size_t buf_len)
     sensory_asr->brick_count++;
     sensory_asr->word_id = -1;
 
-    //uint32_t timer_start = get_reference_time();
-
     error = SensoryProcessData((s16 *) audio_buf, app);
 
-    //uint32_t timer_end = get_reference_time();
-    //asr_printf("Duration: %lu (us)\n", (timer_end - timer_start) / 100);
-
     // if (t->tokensPruned) {
```

Comment:
WOndering if this needs to be deleted or a comment?

Reply:
@ed-xmos, does [6e07070](https://github.com/xmos/sln_voice/pull/299/commits/6e0707031e63d8c840f7314f430c68a6c15d342a) address your concern?